ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGLM.scala

```diff
@@ -62,8 +63,8 @@ trait H2OGLMParams extends H2OAlgoSupervisedParams[GLMParameters] {
   private val solver = stringParam("solver", "solver")
   private val tweedieVariancePower = doubleParam("tweedieVariancePower")
   private val tweedieLinkPower = doubleParam("tweedieLinkPower")
-  private val alpha = nullableDoubleArrayParam("alpha")
-  private val lambda_ = nullableDoubleArrayParam("lambda_", "lambda")
```

Comment:
this `_` is breaking the api style. It was introduced because lambda is keyword on python side, but it is a hack. Rather rename the param. Renaming alpha as to be consistent